client/components/deposits-overview/index.tsx

```diff
@@ -2,86 +2,115 @@
+	// const { includesFinancingPayout } = useDepositIncludesLoan(
+	// 	nextDeposit.id
+	// );
 	const isLoading = isLoadingOverview || isLoadingDeposits;
 
 	// This card isn't shown if there are no deposits, so we can bail early.
-	if ( ! hasNextDeposit && ! isLoading && deposits.length === 0 ) {
+	if ( ! isNextDepositExpected && ! isLoading && deposits.length === 0 ) {
 		return <></>;
 	}
 
 	return (
 		<Card className="wcpay-deposits-overview">
-			<CardHeader>{ strings.heading }</CardHeader>
-			{ /* Only show the next deposit section if the page is loading or if deposits are not blocked. */ }
-			{ ( isLoading || ! account?.deposits_blocked ) && (
-				<>
-					<DepositOverviewSectionHeading
-						title={ strings.nextDeposit.title }
-						text={ strings.nextDeposit.description }
-						isLoading={ isLoading }
-					/>
-					<NextDepositDetails
-						isLoading={ isLoading }
-						overview={ overview }
+			<CardHeader>
+				{ __( 'Deposits', 'woocommerce-payments' ) }
+			</CardHeader>
+
+			{ /* Deposit schedule message */ }
+			{ ! isLoading && isNextDepositExpected && !! account && (
+				<CardBody className="wcpay-deposits-overview__schedule__container">
+					<DepositSchedule
+						depositsSchedule={ account.deposits_schedule }
 					/>
-				</>
+				</CardBody>
 			) }
-			{ /* Only show the deposit history section if the page is finished loading and there are deposits. */ }
-			{ ! isLoading && !! account && !! deposits && deposits.length > 0 && (
-				<>
-					{ account.deposits_blocked ? (
-						<DepositOverviewSectionHeading
-							title={ strings.depositHistoryHeading }
-							children={ <SuspendedDepositNotice /> }
-						/>
+
+			{ /* Notices */ }
+			{ ! isLoading && (
+				<CardBody className="wcpay-deposits-overview__notices__container">
+					{ account?.deposits_blocked ? (
+						<SuspendedDepositNotice />
 					) : (
-						<DepositOverviewSectionHeading
-							title={ strings.depositHistoryHeading }
-							text={
-								<DepositSchedule
-									{ ...account.deposits_schedule }
-								/>
-							}
-						/>
+						<>
+							{ isNextDepositExpected && (
+								<DepositTransitDaysNotice />
+							) }
+							{ /* includesFinancingPayout && <DepositIncludesLoanPayoutNotice /> */ }
+							{ ! hasCompletedWaitingPeriod && (
+								<NewAccountWaitingPeriodNotice />
+							) }
+							{ isNegativeBalanceDepositsPaused && (
+								<NegativeBalanceDepositsPausedNotice />
+							) }
+						</>
 					) }
-					<RecentDepositsList deposits={ deposits } />
-				</>
+				</CardBody>
 			) }
+
+			{ /* Only show the deposit history section if the page is finished loading and there are deposits. */ }
+			{ ! isLoading &&
+				!! account &&
+				!! deposits &&
+				deposits.length > 0 &&
+				! account?.deposits_blocked && (
+					<>
+						<CardBody className="wcpay-deposits-overview__heading">
+							<span className="wcpay-deposits-overview__heading__title">
+								<Loadable
+									isLoading={ isLoading }
```

Comment:
This `Loadable` is unreachable since we're not rendering this if `! isLoading`. Need to either remove the `! isLoading` conditional or remove redundant `Loadable`.